core/src/main/scala/org/apache/spark/sql/rapids/tool/AppBase.scala

```diff
@@ -34,15 +34,15 @@ import org.apache.spark.internal.Logging
 import org.apache.spark.scheduler.{SparkListenerEvent, StageInfo}
 import org.apache.spark.sql.execution.SparkPlanInfo
 import org.apache.spark.sql.execution.ui.{SparkPlanGraph, SparkPlanGraphNode}
-import org.apache.spark.util.{JsonProtocol, Utils}
+import org.apache.spark.util.Utils
 
 abstract class AppBase(
     val eventLogInfo: Option[EventLogInfo],
     val hadoopConf: Option[Configuration]) extends Logging {
 
   var sparkVersion: String = ""
   var appEndTime: Option[Long] = None
-  // The data source information
+  // GPU data source information
```

Comment:
shouldn't be GPU datasources, should be all, reverts

Reply:
done